ncm-fstab/src/main/perl/fstab.pm

```diff
@@ -38,7 +38,7 @@ sub update_entries
         my $el2 = $el->getNextElement();
         my $fs = NCM::Filesystem->new ($el2->getPath()->toString(), $config);
         $self->debug (4, "Update fstab entry at $fs->{mountpoint}");
```

Comment:
@ned21 $fh->close() is only called when no filehandle is passed, so this won't work here. Unless a new fh is created each call..